rclpy/rclpy/node.py

```diff
@@ -1053,12 +1054,18 @@ def set_parameters_callback(
         callback: Callable[[List[Parameter]], SetParametersResult]
     ) -> None:
         """
-        Register a set parameters callback.
+        DEPRECATED. Register a set parameters callback.
```

Comment:
@jacobperron here and elsewhere, I think you may want to use the [`deprecated` directive](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated).

Reply:
Using deprecated directive: 36743c7
Correct me if I'm wrong, but I think there is only this one place to use the directive (at least in the scope of this PR).

Reply:
@hidmic Friendly ping.